staging/src/k8s.io/legacy-cloud-providers/azure/azure_client.go

```diff
@@ -201,7 +204,7 @@ func (az *azVirtualMachinesClient) CreateOrUpdate(ctx context.Context, resourceG
 		klog.V(10).Infof("azVirtualMachinesClient.CreateOrUpdate(%q, %q): end", resourceGroupName, VMName)
 	}()
 
-	mc := newMetricContext("vm", "create_or_update", resourceGroupName, az.client.SubscriptionID, source)
+	mc.Start()
```

Comment:
why did you add this? I don't think this is required

Reply:
We want to measure the time taken for the specific cloud operation. This should not include time from any other items like logging, hence the need to start the timer.